### DISCUSSION TITLE
为什么会频繁的需要重复认证？是需要其他的额外设置吗？

### DISCUSSION THREAD

**Discussion by lovelykid** [XiaoMi/ha_xiaomi_home#463] [Q&A]
经常发现家里的设备无法控制了，然后发现设备都离线了，登录ha 之后发现是小米账户授权到期了，需要重新登录授权，每隔几天都会掉一次，这个有无长期登录的方式？

**Comment by pcr** [comment]
现在的授权有效期确实太短了，好像一周就会过期，对于智能家居而言，个人觉得至少要和MIJIA一样，不然没法放心用。

**Reply by lovelykid** [reply]
是的，用auto 的时候，好几年了，几乎不需要第二次授权。

**Reply by NXY666** [reply]
> 是的，用auto 的时候，好几年了，几乎不需要第二次授权。

毕竟一个出了好几年，一个才刚出几周😂

**Comment by NXY666** [comment]
试一试更新到beta版，授权逻辑有改。

**Reply by lovelykid** [reply]
已更新最新版本，等过几天试试看
